src/index.scss

```diff
@@ -1,6 +1,10 @@
-@import '~@edx/paragon/scss/edx/theme.scss';
-
-@import 'discussions/index.scss';
+@import "@edx/brand/paragon/fonts.scss";
+@import "@edx/brand/paragon/variables.scss";
+@import "@edx/paragon/scss/core/core.scss";
+@import "@edx/brand/paragon/overrides.scss";
 
 @import "~@edx/frontend-component-header/dist/index";
 @import "~@edx/frontend-component-footer/dist/footer";
+
+$fa-font-path: "~font-awesome/fonts";
+@import "~font-awesome/scss/font-awesome";
```

Comment:
Just refactored a bit to match new template.

Reply:
I think we dont need font-awesome